package.json

```diff
@@ -57,6 +57,7 @@
     "eslint": "8.26.0",
     "eslint-config-prettier": "8.5.0",
     "eslint-plugin-prettier": "4.2.1",
+    "fast-xml-parser": "^4.0.15",
```

Comment:
I think this needs to be a prod `dependencies` in `app-builder-lib` specifically.

Reply:
`fast-xml-parser` is only referenced in the unit tests.  So my thought is that it's only a devDependency here

Reply:
ahhh fair enough. Does it still need to be at the electron-builder workspace level or can it be installed within the `/test/package.json` then?

Reply:
I went ahead and moved the dependency to the appropriate location, under test